README.md

```diff
@@ -117,32 +119,33 @@ the desired state of the world.
 Each file in the tree describes a Task.
 
 On the nodeup side, Tasks can manage files, systemd services, packages etc.
-On the `kops create cluster` side, Tasks manage cloud resources: instances, networks, disks etc.
+On the `kops update cluster` side, Tasks manage cloud resources: instances, networks, disks etc.
 
 ## Workaround for terraform bug
 
 Terraform currently has a bug where it can't create AWS tags containing a dot.  Until this is fixed,
 you can't use terraform to build EC2 resources that are tagged with `k8s.io/...` tags.  Thankfully this is only
 the volumes, and it isn't the worst idea to build these separately anyway.
 
-We divide the 'kops create cluster' model into three parts:
-* models/config which contains all the options
+We divide the cloudup model into three parts:
+* models/config which contains all the options - this is run automatically by "create cluster"
 * models/proto which sets up the volumes and other data which would be hard to recover (e.g. likely keys & secrets in the near future)
 * models/cloudup which is the main cloud model for configuring everything else
 
 So you don't use terraform for the 'proto' phase (you can't anyway, because of the bug!):
 
 ```
 export KOPS_STATE_STORE=s3://<somes3bucket>
-export CLUSTER_NAME=<kubernetes.mydomain.com>
-${GOPATH}/bin/kops create cluster --v=0 --zones=us-east-1c --name=${CLUSTER_NAME} --model=config,proto
+export NAME=<kubernetes.mydomain.com>
+${GOPATH}/bin/kops create cluster --v=0 --zones=us-east-1c ${NAME}
```

Comment:
Actually it doesn't need to be... the two changes that are happening here are that `create cluster` only creates a configuration (by default), and thus only uses the config model.  And that `--name` can now be specified as a positional argument.

Going to add a changelog that covers these changes :-)  I'm also going to try to minimize the behavioural changes here as well (and provide more hints from the CLI as to the next steps, so it should be discoverable!)